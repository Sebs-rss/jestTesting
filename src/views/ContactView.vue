// Aqu√≠ crea el componente para la vista Contact

<template>
<div class="contact">
    <img alt="Vue logo" src="../assets/logo.png">
    <h1>Contact</h1>
  </div>
</template>

<script>
    export default {
        name: 'ContactView'
    }
</script>

<style scoped>

</style>