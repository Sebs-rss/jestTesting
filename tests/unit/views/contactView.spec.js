// Ahora creo aquÃ­ las pruebas unitarias para el componente ContactView.vue

import { shallowMount } from '@vue/test-utils'
import ContactView from '@/views/ContactView.vue'

describe('ContactView.vue', () => {
    test('Probando la existencia del componente o vista ContactView ', async () => {
        const wrapper = shallowMount(ContactView)
        expect(wrapper.findComponent(ContactView).exists()).toBe(true)
    }),
    // Prueba de match con el snapshot
    it('matches the snapshot', () => { const wrapper = shallowMount(ContactView); expect(wrapper.html()).toMatchSnapshot();
});
})