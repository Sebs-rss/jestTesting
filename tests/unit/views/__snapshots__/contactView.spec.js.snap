// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView.vue matches the snapshot 1`] = `
<div class="contact"><img alt="Vue logo" src="">
  <h1>Contact</h1>
</div>
`;
